apps/site-landing/src/containers/ReturnProfile/data/params.ts

```diff
@@ -51,7 +47,7 @@ export type SimulationParameters = {
   // How much money flows in each week
   // any monthly income (eg rent, child benefits)
   // Any yearly income (eg bonus's, etc)
-  income: FinancialParams;
+  income: PeriodicalParams;
```

Comment:
I hate naming things...

Reply:
Is that why you rename things so often? 😂